test-py/test_backends_designspace.py

```diff
@@ -222,6 +222,28 @@ async def test_addAnchor(writableTestFont):
     )
 
 
+async def test_read_glyph_locked(writableTestFont):
```

Comment:
You can use the `testFont` fixture here instead: you are not writing to the font after all.

Reply:
right. done.